### PR TITLE
[DSM] PEPPER-1193 fixed grammar error

### PR DESCRIPTION
**Pull request by pegahtah**
## Context

I changed the text that contained a problem (`No kit for participant with ShortId XYZ was not found.`) to `No kit for participant with ShortId XYZ was found.`

## Checklist

- [x] I have labeled the type of changes involved using the `C-*` labels.
- [x] I have assessed potential risks and labeled using the `R-*` labels.
- [x] I have considered error handling and alerts, and added `L-*` labels as needed.
- [x] I have considered security and privacy, and added `I-*` labels as needed
- [x] I have analyzed my changes for stability, fault tolerance, graceful degradation, performance bottlenecks and written a brief summary in this PR.
- [x] If applicable, I have discussed the analytics needs at both a platform and study level with Product and instrumented code accordingly.
- [x] If applicable, my UI/UX changes have passed muster with Product/Design via an over-the-shoulder review, screenshots, etc.

_If unsure or need help with any of the above items, add the `help wanted` label. For items that starts with `If applicable`, if it is not applicable, check it off and add `n/a` in front._

## FUD Score

_Overall, how are you feeling about these changes?_

- [x] :relaxed: All good, business as usual!
- [ ] :sweat_smile: There might be some issues here
- [ ] :scream: I'm sweaty and nervous

## How do we demo these changes?

_How does one observe these changes in a deployed system? Note that **user visible** encompasses many personas--not just patients and study staff, but also ops duty, your fellow devs, compliance, etc._

- [x] They are user-visible in dev as a regular user journey and require no additional instructions.
- [ ] Getting dev into a state where this is user-visible requires some tech fiddling. I have documented these steps in the related ticket.
- [ ] Requires other features before it's human visible. I have documented the blocking issues in jira.
- [ ] I have no idea how to demo this. Please help me!

## Testing

- [ ] I have added regression test labels to the jira ticket
- [ ] I have added verification steps to the jira ticket
- [ ] I have written automated positive tests
- [ ] I have written automated negative tests
- [x] I have written zero automated tests but have poked around locally to verify proper functionality
- [ ] The jira ticket has acceptance criteria and QA has the needed information to test changes

## Release

- [x] These changes require no special release procedures--just code!
- [ ] Releasing these changes requires special handling and I have documented the special procedures in the release plan document

### CODE REVIEW

File: pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/kit/KitInitialScanUseCase.java [hunk 1]
```diff
@@ -33,11 +33,6 @@ private Optional<ScanError> updateKitRequest(String hruid, String kit) {
                 return Optional.ofNullable(new ScanError(kit,
                         "Too many active kits found for \"" + hruid + "\".\n" + UserErrorMessages.IF_QUESTIONS_CONTACT_DEVELOPER));
             }
-
-            if (kitList.size() == 0) {
-                return Optional.ofNullable(new ScanError(kit, "No kit for participant with ShortId \"" + hruid + "\" was not found.\n"
-                        + UserErrorMessages.IF_QUESTIONS_CONTACT_DEVELOPER));
-            }
```
Comment (pegahtah) [inline pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/kit/KitInitialScanUseCase.java:40]: On line 31 ^ we have already checked this so this condition was always false

**Review by Rinchan20 — APPROVED** [review]
looks good to me

File: pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/kit/KitInitialScanUseCase.java [hunk 2]
```diff
@@ -57,7 +52,7 @@ private Optional<ScanError> updateKitRequest(String hruid, String kit) {
                 }
             }
         }
-        return Optional.ofNullable(new ScanError(kit, "No kit for participant with ShortId \"" + hruid + "\" was not found.\n"
+        return Optional.ofNullable(new ScanError(kit, "kit for participant with ShortId \"" + hruid + "\" was not found.\n"
```
Comment (ssettipalli) [inline pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/kit/KitInitialScanUseCase.java]: May be capital K (Kit)

Comment (pegahtah) [reply pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/kit/KitInitialScanUseCase.java]: oh you're right, thank you!

File: pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/kit/KitInitialScanUseCase.java (hunk 1 above)
Comment (ssettipalli) [reply pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/kit/KitInitialScanUseCase.java:40]: If throwing this error is helpful, may be this should go into the else of if on line# 31 
if we dont want to alter the behavior now, thats fine too.

**Review by ssettipalli — APPROVED** [review]

**Review by Rinchan20 — APPROVED** [review]

**Pull request merged**